samples/openapi/README.md

```diff
@@ -0,0 +1,75 @@
+# Open API consumption using low level functions and RAW JSON structures
+
+This example demonstrates how to consume new [OpenAPI](https://vdc-download.vmware.com/vmwb-repository/dcr-public/71f952e6-c14b-417d-8749-dbb5ff2dd48a/9b26a7c0-0cee-40a2-8c01-2f15472324cf/com.vmware.vmware_cloud_director.openapi_34_0.pdf) in VMware Cloud Director. 
+
+OpenAPI low level functions consist of the following to match REST API:
+* OpenApiGetAllItems (FIQL filtering can be applied to narrow down results)
```

Comment:
Maybe there's a VCD doc page where one could read on FIQL? If yes, would be great to have a link here.

Reply:
Sadly nope - I was not able to find one. The only thing I found is -  https://tools.ietf.org/html/draft-nottingham-atompub-fiql-00 although even the link says it is a "draft"